components/tools/OmeroJava/test/integration/ImporterTest.java

```diff
@@ -638,6 +638,27 @@ public void testImportGraphicsImages() throws Exception {
         Assert.assertEquals(0, failures.size());
     }
 
+    @Test(timeOut = 5000)
+    public void testImportInsaneImage() throws Exception {
+        // Simulates QA 17685 (an image with unreasonably huge pixel sizes stuck the import process
+        // in a basically endless loop when trying to throw an exception)
+        File f = new File(
+                System.getProperty("java.io.tmpdir"),
```

Comment:
Maybe `File.createTempFile` or similar?

Reply:
Doesn't work, `File.createTempFile` adds random characters to the file name, but for the 'fake' import it to work it needs an exact file name.

Reply:
Ah, of course, and this class doesn't seem to have an instance of `TempFileManager` handy. Fair enough, no need to right now refactor the whole importer test class to use one.